mdb.py

```
@@ -226,14 +714,728 @@ def execute_change(db, server, sql):
+        # Connect to the database through ProxySQL admin interface
+        db_connect(db, server=server, dictionary=True)
+
+        logging.debug(f"Extracting schema for table: {database}.{table_name} via server: {server}")
+
+        # Use SHOW CREATE TABLE to get the full table definition
+        query = f"SHOW CREATE TABLE `{database}`.`{table_name}`"
+        db['cnf']['servers'][server]['cur'].execute(query)
+        create_table_result = db['cnf']['servers'][server]['cur'].fetchone()
+
+        if not create_table_result:
+            raise ValueError(f"Table '{table_name}' not found in database '{database}'")
```

Ruff (0.15.4)

[warning] Abstract `raise` to an inner function

(TRY301)